packages/react/src/views/QuoteMessage/QuoteMessage.js

```diff
@@ -10,18 +10,71 @@ import {
 import RCContext from '../../context/RCInstance';
 import { useMessageStore } from '../../store';
 import useQuoteMessageStyles from './QuoteMessage.styles';
+import Attachment from '../AttachmentHandler/Attachment';
 
 const QuoteMessage = ({ className = '', style = {}, message }) => {
   const { RCInstance } = useContext(RCContext);
+  const instanceHost = RCInstance.getHost();
   const getUserAvatarUrl = (username) => {
-    const host = RCInstance.getHost();
-    const URL = `${host}/avatar/${username}`;
+    const URL = `${instanceHost}/avatar/${username}`;
     return URL;
   };
   const styles = useQuoteMessageStyles();
   const setQuoteMessage = useMessageStore((state) => state.setQuoteMessage);
 
   const { classNames, styleOverrides } = useComponentOverrides('QuoteMessage');
+
+  // Function to render the message content or attachment preview
+  const renderMessageContent = () => {
```

Comment:
Let's not have a function here like this, try using JSX with required conditions

Reply:
Okay will modify this